pkg/capabilities/triggers/on_demand_trigger.go

```diff
@@ -0,0 +1,77 @@
+package triggers
```

Comment:
Not a huge fan of this file being in chainlink-common. I understand that this is for convenience but longer-term this should end up in a separate repo (or even core seems to be a better fit).

Reply:
I've posted a slack message to discuss in keystone-dev.